src/ks2e-m-dc5-element.ts

```diff
@@ -0,0 +1,308 @@
+import { html, LitElement } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { ElementPin } from './pin';
+
+@customElement('wokwi-ks2e-m-dc5')
+export class KS2EMDC5Element extends LitElement {
+  y1pos = 8.85;
+  y2pos = 70.5;
+  x1pos = 14.7;
+  x2pos = 55;
+  x3pos = 95.3;
+  x4pos = 155.8;
+
+  get pinInfo(): ElementPin[] {
+    return [
+      { name: '1', x: this.x1pos, y: this.y1pos, signals: [], description: '1' },
+      { name: '4', x: this.x2pos, y: this.y1pos, signals: [], description: '4' },
+      { name: '6', x: this.x3pos, y: this.y1pos, signals: [], description: '6' },
+      {
+        name: '8',
+        x: this.x4pos,
+        y: this.y1pos,
+        signals: [{ type: 'power', signal: 'GND' }],
+        description: '8',
+      },
+      { name: '9', x: this.x1pos, y: this.y2pos, signals: [], description: '9' },
+      { name: '11', x: this.x2pos, y: this.y2pos, signals: [], description: '11' },
+      { name: '13', x: this.x3pos, y: this.y2pos, signals: [], description: '13' },
+      { name: '16', x: this.x4pos, y: this.y2pos, signals: [], description: '16' },
```

Comment:
I think it's better to use descriptive pin names: COIL1, COIL2 for the coil pins, then P1 / NO1 / NC1 (normally open / normally closed), P2, NO2, NC2  for the pole pins.

The pin number can go into the "number" field (e.g. number: 16), and the description can be left out (or just "Common Terminal of POLE1", "Normally Closed Terminal of POLE1"), etc.